core/deployment/src/main/java/io/quarkiverse/operatorsdk/deployment/AddRoleBindingsDecorator.java

```diff
@@ -91,37 +113,62 @@ public static String getRoleBindingName(String controllerName) {
         return controllerName + "-role-binding";
     }
 
+    public static String getRandomBindingName(String name) {
```

Comment:
I would rather prefer we used the name of the role ref for the binding name instead of generating a random one.

Reply:
done